backend/monitoring/dashboard.py

```
@@ -666,7 +675,8 @@ async def _get_active_alerts(self) -> List[Dict]:
         
         try:
             # Check system health
-            system_health = await integration_monitor.get_system_health()
+            monitor = get_integration_monitor()
```

Ruff (0.12.2)

Undefined name `get_integration_monitor`

(F821)